tests/test_document_processor.py

```
@@ -295,6 +295,22 @@ async def test_freestyle_rerun_updates_sheet_row_not_append(patched):
     m["append_document_row"].assert_not_called()
 
 
+@pytest.mark.asyncio
+async def test_sheets_task_uses_spawn_background(patched, monkeypatch):
+    """The Sheets write must go through spawn_background — a bare asyncio.create_task()
+    discards its reference and swallows failures silently (architecture review 2026-08-19)."""
+    document, m = patched
```

Ruff (0.16.1)

[warning] Unpacked variable `m` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)